scripts/gameplay/Library.lua

```diff
@@ -35,4 +35,12 @@ UpdateDijkstra = ECS.Component()
 MakeDungeonRequest = ECS.Component()
 Dungeon = {}
 
-FollowBehaviour = ECS.Component{ distance = 5 }
\ No newline at end of file
+FollowBehaviour = ECS.Component{ distance = 5 }
+
+Inventory = ECS.Component{items = {}}
+InventoryEntity = World:Entity(Inventory{items = {}})
```

Comment:
Instead of this, add Inventory to PlayerEntity and any other thing you want to carry items

Reply:
When you want to open the player's inventory, query it directly via `PlayerEntity[Inventory]` and if you want some other inventory, get their entities and do the same :)